docs/grammar/Solidity.g4

```diff
@@ -289,6 +291,18 @@ eventDefinition:
 	Anonymous?
 	Semicolon;
 
+/**
+ * Parameter of an error.
+ */
+errorParameter: type=typeName name=identifier?;
```

Comment:
What about types composed of multiple words, e.g. `error E(address payable _addr)`?

Reply:
`typeName` includes that. I added a test.